### PR TITLE
Add entries to suspicious comment list

### PR DESCRIPTION
**Pull request by rain0r**
Added a few words I got from [owasp.org](https://www.owasp.org/index.php/Review_webpage_comments_and_metadata_for_information_leakage_(OTG-INFO-005)) and [mitre.org](https://cwe.mitre.org/data/definitions/546.html).

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrulesBeta/ZapAddOn.xml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-	<version>15</version>
+	<version>16</version>
 	<status>beta</status>
 	<description>The beta quality Passive Scanner rules</description>
 	<author>ZAP Dev Team</author>
 	<url/>
 	<changes>
 	<![CDATA[
-	Support security annotations for forms that dont need anti-CSRF tokens.<br>
+	Support security annotations for forms that dont need anti-CSRF tokens. <br/>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesBeta/ZapAddOn.xml]: This (old) change entry can be removed.

**Comment by rain0r** [comment]
@thc202 done
@kingthorin done

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
The new version is now available in the marketplace.

**Pull request merged**